platform-hub-api/app/controllers/users_controller.rb

```diff
@@ -49,6 +58,42 @@ def revoke_admin
     head :no_content
   end
 
+  # POST /users/:id/activate
+  def activate
+    keycloak_agent_service.activate_user(@user)
+
+    @user.activate!
+
+    AuditService.log(
+      context: audit_context,
+      action: 'make_active',
+      auditable: @user
+    )
+
+    head :no_content
+  rescue => e
+    render_error "User activation failed - #{e.class}: #{e.message}",
```

Comment:
I notice that the Keycloak Agent service does return specific errors, in which case we should probably catch the individual errors, log them internally and respond with appropriate API response, like [we do elsewhere](https://github.com/UKHomeOffice/platform-hub/blob/3af3078f3e8a5bd6aa83de7820d4673bd960909b/platform-hub-api/app/controllers/identity_flows_controller.rb#L36-L45)?